data-deletion-apis.md

```diff
@@ -2,20 +2,10 @@
 
 ---
 
- * [Deleting Raw Data for a Metric](api/delete-raw.md)
-
- * [Deleting Numeric Rollup Data for a Metric](api/delete-nnt.md)
-
- * [Deleting Text Data for a Metric](api/delete-text.md)
-
- * [Deleting Text Data for a Check](api/delete-text-check.md)
-
- * [Deleting Histogram Data for a Metric](api/delete-histogram.md)
-
- * [Deleting Histogram Data for a Check](api/delete-histogram-check.md)
-
- * [Deleting Raw Data for a Metric](api/delete-raw.md)
-
- * [Deleting All Data Prior to a Date (Sweep Delete)](api/delete-sweep.md)
- 
-   * [Cancelling a Sweep Delete](api/delete-sweep-cancel.md)
+Deletion APIs can be broken down into three categories, depending on the deletion scenario:
+## Deleting data for a metric or group of metrics of a particular data type up to a specified end time (datatype delete)
+This is intended for metrics that will be remaining in use, but the user just wants to remove older data that is no longer needed. [Click here for details.](data-deletion-datatype.md)
+## Deleting all metric data of all types up to a specified end time (sweep delete)
+This is intended for removing all data older than the specified end time for all metrics of any data type, as part of storage hygeine and/or data retention policies. [Click here for details.](data-deletion-sweep.md)
+## Deleting all metric data and metadata for a metric or group of metrics (full delete)
```

Comment:
Now that I can see this stuff rendered, these long headings are harder to read, due to the large font and resulting line wrapping.  Perhaps the headings themselves could be the shorter description you currently have in parentheses, e.g. "Data-Type Delete" with the rest of the current heading being the first sentence of the section, followed by the "this is intended..." bit.  IMO it's easier to scan short headings for what looks interesting rather than reading a sentence that is rendered as a heading.

Reply:
ok sure